packages/dashboard/src/components/internalDashboard/index.tsx

```diff
@@ -230,86 +248,110 @@ const InternalDashboard: React.FC<InternalDashboardProperties> = ({ onSave, edit
   }
 
   const EditComponent = (
-    <div className='dashboard' style={userSelect}>
-      <CustomDragLayer onDrag={(isDragging) => setUserSelect(isDragging ? disabledUserSelect : defaultUserSelect)} />
-      <div style={dashboardToolbarBottomBorder} className='dashboard-toolbar'>
-        <Box float='left' padding='xs'>
-          <ComponentPalette />
-        </Box>
-        <Box float='right' padding='xs'>
-          <SpaceBetween size='s' direction='horizontal'>
-            <TimeSelection isPaginationEnabled />
-            <Divider key='2' />
-            <Actions
-              key='3'
-              readOnly={readOnly}
-              onSave={onSave}
-              dashboardConfiguration={dashboardConfiguration}
-              grid={grid}
-              significantDigits={significantDigits}
-              editable={editable}
-            />
-          </SpaceBetween>
+    <ContentLayout
+      disableOverlap
+      header={
+        <Box padding={{ horizontal: 's', top: 'm' }}>
```

Comment:
let's make a component for dashboard header.

Reply:
Done